packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1558,7 +1558,19 @@ const useEchartsCartesianConfig = (
             formatter: (params) => {
                 if (!Array.isArray(params) || !itemsMap) return '';
 
+                const flipAxes = validCartesianConfig?.layout.flipAxes;
+                const getTooltipHeader = () => {
+                    if (flipAxes && !('axisDim' in params[0])) {
```

Comment:
This is because the properties are different if you hover a bar or a group...

![Screenshot from 2024-04-17 17-11-25](https://github.com/lightdash/lightdash/assets/1983672/dad5933b-917f-429c-a5f0-f05cb610475d)
![Screenshot from 2024-04-17 17-11-41](https://github.com/lightdash/lightdash/assets/1983672/3cab1a6e-943b-4756-b984-0099fd602e8a)